NGitLab.Tests/IssueTests.cs

```diff
@@ -226,5 +227,32 @@ public async Task Test_get_all_resource_label_events()
             Assert.AreEqual(testLabel, removeLabelEvent.Label.Name);
             Assert.AreEqual(ResourceLabelEventAction.Remove, removeLabelEvent.Action);
         }
+
+        [Test]
+        [NGitLabRetry]
+        public async Task Test_get_new_and_updated_issue_with_duedate()
+        {
+            using var context = await GitLabTestContext.CreateAsync();
+            var project = context.CreateProject();
+            var issuesClient = context.Client.Issues;
+            const string formatString = "yyyy-MM-dd";
+
+            var initialDueDate = DateTime.Now.AddDays(3).ToString(formatString);
```

Comment:
I don't think you need to use variable date (DateTime.Now). Instead, you can hardcode a date using `new DateTime(2022, 01, 01)` to reduce the chance of flakiness.

Reply:
For some reason I just assumed the API would reject a date in the past 🤦 updated to use hardcoded dates.